hoot-core-test/src/test/cpp/hoot/core/TestUtils.h

```diff
@@ -131,6 +135,17 @@ class TestUtils
   static ElementPtr getElementWithTag(OsmMapPtr map, const QString tagKey, const QString tagValue);
 
   /**
+   * Return the singleton instance.
+   */
+  static shared_ptr<TestUtils> getInstance();
+
+  /**
+   * Register a way to reset the environment. This is most useful in plugins to avoid circular
+   * references.
+   */
+  void registerReset(RegisteredReset* r) { _resets.append(r); }
```

Comment:
This is the only spot.

https://github.com/ngageoint/hootenanny/blob/82/hoot-rnd/src/test/cpp/hoot/rnd/conflate/network/NetworkVertexTest.cpp#L35

For this to work without registering I'd have to add a check in TestUtils that tries to instantiate NetworkVertex and then register it which would create a circular runtime reference between core and rnd.